backends/p4tools/modules/testgen/targets/bmv2/table_stepper.cpp

```diff
@@ -96,8 +91,10 @@ const IR::Expression *Bmv2V1ModelTableStepper::computeTargetMatchType(
             maxKey = IR::getConstant(keyExpr->type, IR::getMaxBvVal(keyExpr->type));
             keyExpr = minKey;
         } else {
-            minKey = ToolsVariables::getSymbolicVariable(keyExpr->type, minName);
-            maxKey = ToolsVariables::getSymbolicVariable(keyExpr->type, maxName);
+            auto symbolicTableRange = Bmv2ControlPlaneState::getTableRange(
+                properties.tableName, keyProperties.name, keyExpr->type);
+            minKey = symbolicTableRange.first;
+            maxKey = symbolicTableRange.second;
```

Comment:
Suggestion: use structured bindings

```suggestion
            auto [minKey, maxKey] = Bmv2ControlPlaneState::getTableRange(
                properties.tableName, keyProperties.name, keyExpr->type);

Reply:
The issue is that minKey and maxKey are declared previously. Do structured bindings support overriding of preexisting variables?

Reply:
No, in that case you could probably use `std::tie(minKey, maxKey) = ...`.

Reply:
Didn't know about `std::tie`, fixed! There is always something new to learn about C++. :)

Reply:
Yes, one is always learning with C++ :-). Interesting `std::tie` is the old way of doing this, most of its used can be replaced by structured bindings, but sometimes it is still useful. It is a little sad each of these look so differently syntax-wise.